tests/test_widgets.py

```diff
@@ -185,6 +185,34 @@ class W(SuffixedMultiWidget):
         }, {}, 'price')
         self.assertEqual(result, ['1', 'lt'])
 
+    def test_value_omitted_from_data(self):
+        class A(SuffixedMultiWidget):
+            suffixes = ['_b']
+
+        a = A(widgets=[BooleanWidget])
```

Comment:
Judging from https://github.com/carltongibson/django-filter/blob/5d1f6a512c0c8f092897bf805a3b3990451202e8/django_filters/widgets.py#L108 I think we want an instance of `BooleanWidget` here, not a class: the widget's `value_omitted_from_data` is a regular method that wants `self`.

Reply:
The `MultiWidget` base instantiate the widgets if provided as classes. So, there should be no issue with accidentally passing a class instead. See:

https://github.com/django/django/blob/d232fd76a85870daf345fd8f8d617fe7802ae194/django/forms/widgets.py#L804

Reply:
@rpkilby thanks for clarifying this. My intention was to keep the tests within `SuffixedMultiWidgetTests` consistent; hence I decided to stay with passing the class, instead of passing an instance.